src/openms/source/CHEMISTRY/NucleicAcidSpectrumGenerator.cpp

```diff
@@ -351,8 +346,7 @@ namespace OpenMS
     for (Size i = 0; i < size; ++i)
     {
       spectrum.push_back(uncharged_spectrum[i]);
-      spectrum.back().setMZ(abs(spectrum.back().getMZ() / charge +
-                                Constants::PROTON_MASS_U));
+      spectrum.back().setMZ(abs(spectrum.back().getMZ() / charge + Constants::PROTON_MASS_U));
```

Comment:
don't use abs because of ambiguity with https://cplusplus.com/reference/cstdlib/abs/
use std::abs instead

Reply:
Should I do that for all the instances of abs() in the old code as well?

Reply:
yes it is a common source of errors

Reply:
Actually it looks like there's just one other instance in this file where `abs()` is run on a floating point value. I'll just go ahead and clarify that one.